core/src/main/java/io/questdb/griffin/engine/table/ShowAllTablePartitionsCursoryFactory.java

```diff
@@ -0,0 +1,321 @@
+            private long partitionCount;
+            private long rowCount;
+            private long sizeB;
+            private boolean walEnabled;
+
+            @Override
+            public void close() throws IOException {
+
+            }
+
+            @Override
+            public long getLong(int col) {
+                switch (col) {
+                    case 3:
+                        return partitionCount;
+                    case 4:
+                        return rowCount;
+                    case 5:
+                        return sizeB;
+                    default:
+                        throw new UnsupportedOperationException();
+                }
+            }
+
+            @Override
+            public boolean getBool(int col) {
+                switch (col) {
+                    case 1:
+                        return walEnabled;
+                    default:
+                        throw new UnsupportedOperationException();
+                }
+            }
+
+            @Override
+            public @Nullable CharSequence getStrA(int col) {
+                switch (col) {
+                    case 0:
+                        return tableName;
+                    case 2:
+                        return PartitionBy.toString(partitionBy);
+                    default:
+                        throw new UnsupportedOperationException();
+                }
+            }
+
+
+            private void loadPartitionDetailsForCurrentTable(TableToken tableToken) {
+                rowCount = 0;
+                partitionCount = 0;
+                sizeB = 0;
+                tableReader = executionContext.getReader(tableToken);
+                TxReader tableTxReader = tableReader.getTxFile();
+                int partitionIndex = 0;
+                tableName = tableToken.getTableName();
+                walEnabled = tableToken.isWal();
+                partitionBy = tableReader.getPartitionedBy();
+                path.of(cairoConfig.getRoot()).concat(tableToken).$();
+                scanDetachedAndAttachablePartitions();
+                rootLen = path.size();
+                int attachedPartitions = tableTxReader.getPartitionCount();
+                partitionCount = attachedPartitions +
+                        attachablePartitions.size() +
+                        detachedPartitions.size();
+
+                while (partitionIndex < partitionCount) {
+                    path.trimTo(rootLen).$();
+                    if (partitionIndex < attachedPartitions) {
+                        long timestamp = tableTxReader.getPartitionTimestampByIndex(partitionIndex);
+                        TableUtils.setPathForPartition
+                                (path, partitionBy, timestamp, tableTxReader.getPartitionNameTxn(partitionIndex));
+                        rowCount += tableTxReader.getPartitionSize(partitionIndex);
+                    } else {
+                        int idx = partitionIndex - attachedPartitions;
```

Comment:
Detached and attachable partitions are useful to see when there is a break down on partition level. On table level, it only makes it confusing, e.g. table row count should not include the row count from detached partitions.

Please remove the logic to scan detached and attachable partitions.